src/main/java/com/autotune/utils/MetricsConfig.java

```diff
@@ -56,6 +58,9 @@ private MetricsConfig() {
         timerBAddPerfProfileDB = Timer.builder("kruizeDB").description(DB_METRIC_DESC).tag("method", "addPerformanceProfileToDB");
         timerBLoadPerfProfileName = Timer.builder("kruizeDB").description(DB_METRIC_DESC).tag("method", "loadPerformanceProfileByName");
         timerBLoadAllPerfProfiles = Timer.builder("kruizeDB").description(DB_METRIC_DESC).tag("method", "loadAllPerformanceProfiles");
+        timerBAddMetricProfileDB = Timer.builder("kruizeDB").description(DB_METRIC_DESC).tag("method", "addMetricProfileToDB");
```

Comment:
Please exclude these metrics. There's no need to track them at the moment. If necessary, we can create a separate PR later.

Reply:
done